src/Item.cpp

```diff
@@ -0,0 +1,52 @@
+//
+// Created by Jahrr on 4/6/2023.
+//
+// Last edited by Jahrr on 4/6/2023
+//
+#include "../include/sudoManager/Item.hpp"
```

Comment:
```suggestion
#include "sudoManager/Item.hpp"
```
No need for a relative path.
`/src/CmakeLists.txt` will handle that